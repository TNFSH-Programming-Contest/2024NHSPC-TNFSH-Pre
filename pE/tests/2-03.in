20
956652 955892 930632 912123 956241 959792 959268 994806 981295 904230 953826 979721 946049 974319 960143 988861 999894 908623 914691 910971 1
