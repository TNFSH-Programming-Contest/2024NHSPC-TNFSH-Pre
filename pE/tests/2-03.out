15414941143063